### ISSUE TITLE
Tab label aligning vertically in some devices when render single tab.

### ISSUE THREAD

**Issue by rifad4u** [react-community/react-navigation#11083]
### Current behavior

When rendering only one tab the tab bar label aligning vertically in some android devices , this is happening when set 'width' : 'auto' and 'tabBarScrollEnabled' : true .

Device details :  OPPO F17 Pro , Android version  : 12

![Screenshot_2022-12-08-11-48-55-93_aa2c0c53276997288a36df793bc3d2c8](https://user-images.githubusercontent.com/37132763/206395879-beb4a826-2879-4910-b98f-c289e22214f9.jpg)

### Expected behavior

Tab bar label should align horizontally.

### Reproduction

https://snack.expo.dev/4zlAqjyuZ

### Platform

- [X] Android
- [ ] iOS
- [ ] Web
- [ ] Windows
- [ ] MacOS

### Packages

- [ ] @react-navigation/​bottom-tabs
- [ ] @react-navigation/​drawer
- [ ] @react-navigation/​material-bottom-tabs
- [x] @react-navigation/​material-top-tabs
- [ ] @react-navigation/​stack
- [ ] @react-navigation/​native-stack
- [x] react-native-tab-view
- [ ] flipper-plugin-react-navigation

### Environment

- [] I've removed the packages that I don't use

| package                                | version |
| -------------------------------------- | ------- |
| @react-navigation/native               |  6.1.0 
| @react-navigation/material-top-tabs    | 6.5.0
| react-native-screens                   | 3.18.2
| react-native-gesture-handler           | 2.8.0
| react-native-reanimated                | 2.13.0
| react-native-tab-view                  | 3.3.3
| react-native-pager-view                |  6.1.0
| react-native                           | 0.70.6

**Comment by okwasniewski** [comment]
Hey, this doesn't seem to be reproducible on the snack you provided. Can you check if this occurs only on your specific version of Android?

**Comment by rifad4u** [comment]
> Hey, this doesn't seem to be reproducible on the snack you provided. Can you check if this occurs only on your specific version of Android?

Yes , This issue happening in specific devices only, I have encountered this issue in OPPO F17 Pro with Android 12 version.

**Comment by Rakxhit** [comment]
i faced the same issue in Samsung Galaxy A14 5G, android version 14, 
i solved it giving the minWidth value in the styles,
